runtimeconfig/unit_tests/test_config.py

```diff
@@ -0,0 +1,326 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from google.cloud._helpers import _rfc3339_to_datetime
+from google.cloud.runtimeconfig._helpers import config_name_from_full_name
+
+
+class TestConfig(unittest.TestCase):
+    PROJECT = 'PROJECT'
+    CONFIG_NAME = 'config_name'
+    CONFIG_PATH = 'projects/%s/configs/%s' % (PROJECT, CONFIG_NAME)
+
+    def _getTargetClass(self):
+        from google.cloud.runtimeconfig.config import Config
+        return Config
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
+    def _verifyResourceProperties(self, config, resource):
+        if 'name' in resource:
+            self.assertEqual(config.full_name, resource['name'])
+            self.assertEqual(
+                config.name,
+                config_name_from_full_name(resource['name']))
+        if 'description' in resource:
+            self.assertEqual(config.description, resource['description'])
+
+    def test_ctor(self):
+        client = _Client(project=self.PROJECT)
+        config = self._makeOne(name=self.CONFIG_NAME,
+                               client=client)
+        self.assertEqual(config.name, self.CONFIG_NAME)
+        self.assertEqual(config.project, self.PROJECT)
+        self.assertEqual(config.full_name, self.CONFIG_PATH)
+
+    def test_ctor_w_no_name(self):
+        client = _Client(project=self.PROJECT)
+        config = self._makeOne(name=None, client=client)
+        self.assertRaises(ValueError, lambda: config.full_name)
```

Comment:
`getattr(config, 'full_name')` would've been preferred.

Do you recall the lint error?